src/ECModel.jl

```diff
@@ -852,6 +853,267 @@ function split_financial_terms(ECModel::AbstractEC, profit_distribution=nothing)
+
+    # Create a vector of years from 2023 to (2023 + project_lifetime)
+    gen_data = ECModel.gen_data
+    project_lifetime = field(gen_data, "project_lifetime")
+
+    business_plan = split_yearly_financial_terms(ECModel)
+    year_set = business_plan.year_set
+
+    # Create an empty DataFrame
+    df_business = DataFrame(Year = Int[], CAPEX = Float64[], OEM = Float64[], EN_SELL = Float64[], EN_CONS = EN_SELL = Float64[], PEAK = Float64[], REP = Float64[], 
+    REWARD = Float64[], RV = Float64[])
+    for i in year_set
+        Year = 0 + year_set[i+1]
+        CAPEX = sum(business_plan.CAPEX[i, :])
+        OEM = sum(business_plan.OEM[i, :])
+        EN_SELL = sum(business_plan.EN_SELL[i, :])
+        EN_CONS = sum(business_plan.EN_CONS[i, :])
+        PEAK = sum(business_plan.PEAK[i, :])
+        REP = sum(business_plan.REP[i, :])
+        REWARD = sum(business_plan.REWARD[i, :])
+        RV = sum(business_plan.RV[i, :])
+        push!(df_business, (Year, CAPEX, OEM,  EN_SELL, EN_CONS, PEAK, REP, REWARD, RV))
+    end
+
+    return df_business
+end
+
+"""
+    business_plan_plot(ECModel::AbstractEC, profit_distribution)
+
+Function to describe the cost term distributions by all users for all years.
+
+Parameters
+----------
+- ECModel : AbstractEC
+    EnergyCommunity model
+- df_business
+    Dataframe with the business plan information
+
+Returns
+-------
+    The output value is a plot with the business plan information
+"""
+
+function business_plan_plot(ECModel::AbstractEC, df_business=nothing)
+    if df_business === nothing
+        df_business = business_plan(ECModel)
+    end
+
+    # Extract the required columns from the DataFrame
+    years = df_business.Year
+    capex = -df_business.CAPEX
+    oem = -df_business.OEM
+    en_sell = df_business.EN_SELL
+    en_cons = -df_business.EN_CONS
+    rep = -df_business.REP
+    reward = df_business.REWARD
+    rv = df_business.RV
+    peak = -df_business.PEAK
+
+    # Create a bar plot
+    p = bar(years, [capex, oem, en_sell, en_cons, rep, reward, rv, peak],
+            label=["CAPEX" "OEM" "Energy sell" "Energy consumption" "Replacement" "Reward" "Recovery" "Peak charges"],
+            xlabel="Year", ylabel="Amount [€]",
+            title="Business Plan Over 20 Years",
+            #ylims=(maximum([capex; oem; en_cons; rep; peak]), maximum([oem; en_sell; reward; rv])*1.2),
+            legend=:bottomright,
+            color=:auto,
+            xrotation=45,
+            bar_width=0.6,
+            grid=false,
+            framestyle=:box,
+            barmode=:stack,
+            )
+
+    #p = @df_business df_business bar(:Year, [:CAPEX, :OEM, :EN_SELL, :EN_CONS, :REP, :REWARD, :RV, :PEAK], xlabel="Year", ylabel="Value", title="Business plan information", bar_position=:stacked, bar_width=0.5, color=[:red :blue :green :orange :purple :yellow :brown :pink], legend=:topleft)
+
+    print(df_business)
+    savefig(p, joinpath("results","Img","business_plan","CO")) # Save the plot

```

Comment:
Remove these lines.
I understand they are meant for debug now.
The plot can be save externally of the function if the user desires that